src/Projects/Modules/Hsnr/Server/Module.Hsnr.Cida/Module.cs

```diff
@@ -7,26 +7,25 @@
 using Module.Hsnr.Extensions;
 using Module.Hsnr.Timetable;
 using Module.Hsnr.Timetable.Data;
-using Module.Hsnr.Timetable.Parser;
-using CalendarType = Hsnr.CalendarType;
-using SemesterType = Module.Hsnr.Timetable.Data.SemesterType;
 
 namespace Module.Hsnr
 {
     public class Module : IModule
     {
-        public void Load()
```

Comment:
That is correct, bc the correct handling of this module is not a part of this PR. For testing purpose i wanted to test this module but our wonderfull university thought it would be a great idea to just block of external connections to the timetable so the initialization of this module will fail due to a http timeout bc it can't reach the server while not in the correct network